### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by matthiasbeyer**
This PR fixes some typos. Please review & ACK.

### CODE REVIEW

File: paper/overview.tex [hunk 1]
```diff
@@ -8,10 +8,10 @@ \section{Overview}
 
     This paper will discuss the algorithms used to implement each operation,
     drawing lines between the different levels, the overall hash-table and the
-    buckets that is. Each operation's description will also include a short
-    derivation of the operation's class of complexity.
+    buckets that is. Each operations description will also include a short
+    derivation of the operations class of complexity.
```
Comment (neithernut) [inline paper/overview.tex:13]: It's a description of the operation, hence the operation's description, same for the second line.

Comment (matthiasbeyer) [inline paper/overview.tex:13]: But in english, you use `'s` as shortcut for `is`. So after expanding, the upper says:

`Each operation is description will also include...`

Which isn't right, is it?

**Comment by neithernut** [comment]
It depends on the context. An apostrophe, followed by an "s" can be either an "it" or an indicator of possession or affliliation.

**Comment by matthiasbeyer** [comment]
Please re-add this comment inside the diff discussion and delete the old one. I will delete this comment afterwards.

File: paper/overview.tex (hunk 1 above)
Comment (TheNeikos) [inline paper/overview.tex:13]: 's is still right here, it merely means that the description belongs to the operation. It doesn't always expand to `is`

Comment (matthiasbeyer) [inline paper/overview.tex:13]: Ok. Will merge then.

**Comment by matthiasbeyer** [comment]
Wha shit, I'm to tired this morning. I will submit a revert-patch!

**Pull request merged**